### PR TITLE
Add requirements for Arch.

### PR DESCRIPTION
**Pull request by Hasaber8**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,49 @@
...
+pacman -S unace unrar zip unzip p7zip sharutils uudeview arj cabextract file-roller
+pacman -S python python-pip brotli lz4
+pip install backports.lzma protobuf pycrypto
 ```
+### For "rename" and "mpack" you need to manually clone and install the packages
+- rename
+```bash
+git clone https://aur.archlinux.org/rename.git
+cd rename 
+makepkg -Acs
```
Comment (danielgusvt) [inline README.md]: Why not just `makepkg -si`? you can avoid the next line

File: README.md [hunk 2]
```diff
@@ -1,28 +1,49 @@
...
+sudo pacman -U rename-1.3-7-x86_64.pkg.tar.xz
+```
+- mpack
+```bash
+git clone https://aur.archlinux.org/mpack.git
+cd mpack
+makepkg -Acs
```
Comment (baalajimaestro) [inline README.md]: A better option would be,
```bash
for package in mpack rename; do
    git clone https://aur.archlinux.org/"${package}"
    cd "${package}" || continue
    makepkg -si --skippgpcheck
    cd - || break
    rm -rf "${package}"
done

**Review by baalajimaestro — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Hasaber8) [reply README.md]: It even installs the package on its own?

Comment (Hasaber8) [reply README.md]: > It even installs the package on its own?

It does , noice af

File: README.md (hunk 1 above)
Comment (Hasaber8) [reply README.md]: I didnt read the full makepkg documentation xD

**Pull request merged**